quality-metrics/new/qm-000-vendor_supported.md

```diff
@@ -0,0 +1,40 @@
+---
+SMQM: UNASSIGNED
+Author: Matt Ray <matt@chef.io>
+Status: Draft
+License: Apache 2.0
+---
+
+# The cookbook is supported by the vendor
+
+The vendor of the product or project managed by the cookbook acknowledges and
+accepts support responsibilities for the cookbook.
+
+* The vendor is listed as a contact email address for the cookbook in the README
+* Chef's Partner Engineering is listed as a contact email address for the cookbook in the README
+* Supermarket represents the cookbook as "Vendor Supported" in some form (badge?)
```

Comment:
This is likely separate from the quality metric.  It's probably best as an issue / feature request for chef/supermarket.  I could see a new "partner manager" role on the Supermarket that gives the users in the role the ability to mark a cookbook as 'vendor supported' or something along those lines.  That mark could be used for searching and for display.